site/ja/tutorials/text/word_embeddings.ipynb

```diff
@@ -0,0 +1,665 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "pHLcFtn5Wsqj"
+   },
+   "outputs": [],
+   "source": [
+    "embedding_dim=16\n",
+    "\n",
+    "model = keras.Sequential([\n",
+    "  layers.Embedding(encoder.vocab_size, embedding_dim),\n",
+    "  layers.GlobalAveragePooling1D(),\n",
+    "  layers.Dense(1, activation='sigmoid')\n",
+    "])\n",
+    "\n",
+    "model.summary()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "JjLNgKO7W2fe"
+   },
+   "source": [
+    "### モデルのコンパイルと訓練"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "lCUgdP69Wzix"
+   },
+   "outputs": [],
+   "source": [
+    "model.compile(optimizer='adam',\n",
+    "              loss='binary_crossentropy',\n",
+    "              metrics=['accuracy'])\n",
+    "\n",
+    "history = model.fit(\n",
+    "  train_batches,\n",
+    "  epochs=10,\n",
+    "  validation_data=test_batches, validation_steps=20)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "LQjpKVYTXU-1"
+   },
+   "source": [
+    "このアプローチにより、モデルの評価時の正解率は 88% 前後に達します（モデルは過学習しており、訓練時の正解率の方が際立って高いことに注意してください）。"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "0D3OTmOT1z1O"
+   },
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "history_dict = history.history\n",
+    "\n",
+    "acc = history_dict['accuracy']\n",
+    "val_acc = history_dict['val_accuracy']\n",
+    "loss = history_dict['loss']\n",
+    "val_loss = history_dict['val_loss']\n",
+    "\n",
+    "epochs = range(1, len(acc) + 1)\n",
+    "\n",
+    "plt.figure(figsize=(12,9))\n",
+    "plt.plot(epochs, loss, 'bo', label='Training loss')\n",
+    "plt.plot(epochs, val_loss, 'b', label='Validation loss')\n",
+    "plt.title('Training and validation loss')\n",
+    "plt.xlabel('Epochs')\n",
+    "plt.ylabel('Loss')\n",
+    "plt.legend()\n",
+    "plt.show()\n",
+    "\n",
+    "plt.figure(figsize=(12,9))\n",
+    "plt.plot(epochs, acc, 'bo', label='Training acc')\n",
+    "plt.plot(epochs, val_acc, 'b', label='Validation acc')\n",
+    "plt.title('Training and validation accuracy')\n",
+    "plt.xlabel('Epochs')\n",
+    "plt.ylabel('Accuracy')\n",
+    "plt.legend(loc='lower right')\n",
+    "plt.ylim((0.5,1))\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "KCoA6qwqP836"
+   },
+   "source": [
+    "## 学習した埋め込みの取得\n",
+    "\n",
+    "次に、訓練によって学習された単語埋め込みを取得してみます。これは、shape が `(vocab_size,embedding-dimension)` の行列になります。"
```

Comment:
[weak suggestion] One whitespace between the words: `(vocab_size,embedding-dimension)`

Reply:
Thanks. Fixed.